### ISSUE TITLE
FAQ needs easy way to contact FOLA if question not answered

### ISSUE THREAD

**Issue by yiminng** [hackforla/food-oasis#1288]
### Problem: 
Both Desktop and Mobile Sites
If I cannot find the answer to my question on the FAQs page, I would expect to find the contact information for further inquiries to appear close by. However, I have to look through the page to realize there is no contact information and then hunt for it on another page (currently it is in the "[About Us](https://foodoasis.la/about)" page ) 

Steps to repro: 
goto foodoasis.la
goto FAQs

Heuristics Violated:
4. Consistency and Standards
10. Help and Documentation

Severity Level 3 as described in [the heuristics spreadsheet](https://docs.google.com/spreadsheets/d/1UCoj5ecVQhOBAPNFvzOyySSeJFNtJ34Bx6Ti2cnLma0/edit#gid=0)

### Recommendations: 
Consider putting contact information on the FAQs page, to the effect of “Something else?” or “Need more help?” 

### Action Items

- [x] Discuss with team if we want change title of "FAQs" to "Contact US" with contact info or add contact us to FAQ page (and keep it also in "About Us")
- [x] design changes
- [ ] dev changes 

This is issue was created from a list of food seeker [issues identified](https://docs.google.com/spreadsheets/d/1UCoj5ecVQhOBAPNFvzOyySSeJFNtJ34Bx6Ti2cnLma0/edit?usp=sharing) by a [heuristic evaluation](https://docs.google.com/presentation/d/1mAQV8V9y1ZoQAgBb_mflU8UAGcRogsVfEgZU-hKVvQ0/edit?usp=sharing) completed by
[@Gigi P](https://hackforla.slack.com/team/U02KZ8WSHHV) & @ryu-jieun at the end of 2021

Note: 
Figma File
https://www.figma.com/file/D3oq9QOXl0rFkwrEUJbABs/FOLA-Design-%231?type=design&node-id=2943-7818&t=VUOMnAsRtXA9qYFL-0

**Comment by staceyrebekahscott** [comment]
@sei1122 @fancyham How difficult is it to make changes to the text on the website? And am I assessing the time accurately when I label this as Size: 2pt?

**Comment by staceyrebekahscott** [comment]
Moving this to Prioritized Backlog as Heuristics is a metric I have made a priority for the Impact Sprints 2022.

**Comment by sei1122** [comment]
Changing text is not hard. 
But do we have Food Oasis email and does anyone check regularly?

**Comment by staceyrebekahscott** [comment]
@sei1122 Yes, the emails get forwarded to the PMs so it would be seen if someone submits a question. 

What would be the workflow for making changes to the text on the site?

**Comment by sei1122** [comment]
My suggestion is 
- Create a Contact US page. This is the easiest for the user as long as PM or someone takes care of the email. If we do this, remove the Contact section from the About section to reduce redundancy. 

The above comment was suggested to change the FAQ to Contact but the FAQ contact is not fit the Contact title.

**Comment by fancyham** [comment]
@itserindean tagging you here — could you chat with Fern to see how 'contact us' emails come in and if/how we might manage them?

@fnkdesign could you add screenshot of your latest designs for discussion?  Thanks!

**Comment by fnkdesign** [comment]
According to the previous discussion, we have our "Contact Us" section on the "[About Us](https://foodoasis.la/about)" page, which makes it complicated for users to jump from "[FAQs](https://foodoasis.la/faqs)" to find our contact information on "[About Us](https://foodoasis.la/about)" page if they need to contact us for additional information.

I agree with @sei1122 suggestion that we can create a specific page for users to contact us. This is more straightforward and easier for users to find a way to contact us. 

So, I designed a "Contact Us" page which included with 2 sections;
1.) A section for users to **find the information they need before reaching out to us**. This will filter out users that trying to ask questions that we already have answers to on the FAQs page. 
2.) A contact form section. Users don't have to log in from their email to contact us and the information from this form will be sent directly to our email (foodoasisinfo@hackforla.org).

![contact us](https://user-images.githubusercontent.com/113713802/193639556-bf628f1d-87fc-44e6-9862-0d1c51ad4e61.jpg)

@itserindean please let us know what you think/feel about this, if there's anything you want to add on, or any concerns?

**Comment by itserindean** [comment]
This design seems to envelop the FAQ but i dont think it would occur to me to click on "Contact us" if i'm looking for FAQ or am i misunderstanding the design?

I'm not sure the underlying premise that users expect contact info as part of the FAQ is accurate. I personally prefer having it as a part of About Us and maybe putting a link at the bottom of FAQ to that page if we think it necessary (I"m not sures it is but don't have any objections to such a link either...)

**Comment by fancyham** [comment]
@itserindean General question: How often do we get emails currently and what types do we typically get?  

Trying to judge how many contacts we get and our capacity for handling them (if we do), and then how to handle contacting us.  Is the current link enough?

![Screen Shot 2022-10-11 at 5 43 36 PM](https://user-images.githubusercontent.com/3376957/195223507-2c0ac0b4-0625-43ab-b42d-a25abd87a2b4.jpg)

I believe we have a separate issue looking to improve the FAQ and actually answer questions that or visitors may ask… it's completely missing things that our primary audience (food-seekers) would be asking.

@fnkdesign Could you also contact the programmer folks in #fola-dev and ask them if or what kinds of email contact forms we might be able to on our site and level of effort for different options?  

It's probably possible to do a web-based email creation form but sometimes there's some behind-the-scenes setup required.

**Comment by itserindean** [comment]
@fancyham We get emails daily but I'd estimate **less than one a month from someone finding our email on the site.**

what we do get:
- There's probably 1-2 emails a week that require a response from me (OF all the email to that address)
- accept/declines for meeting requests
- we're on a few random mailing lists

**Comment by fernksk** [comment]
@fancyham It’s interesting that you mentioned the footer. I’m not sure why we decided to remove it but maybe we should considering bring it back? Since majority of websites still incorporate that and it's there for the purpose of improves the overall usability.

@itserindean According to previous design meetings, we decided to go with "create a proper contact us page" idea. The design of the page has changed from having 2 sections on one page (the design above) to just a form. I added the "confirmation screen" pop up to enhance the user experience as well.

<img width="889" alt="contatc us form" src="https://user-images.githubusercontent.com/72167626/211985538-c12268ff-d177-4a17-a0e3-36bbb799e634.png">
<img width="866" alt="confirmation pop up" src="https://user-images.githubusercontent.com/72167626/211986039-b8f19877-e368-4876-9c03-1e85d6b10b01.png">

Also, from the latest design conversation (before holidays) we talked about the idea of having a "confirmation email" send to users once the form has submitted. We also talked about what kind of content should be on the "confirmation email" either just plain text or possibly image (our logo) + text (if developers can support). 

@fancyham I haven't heard back from the dev team about the form. 
I'll reach out directly to the development team member to discuss on the form itself and the "confirmation email" later this week. 

Once I got the feedback from the development team, we can bring that to the design conversation and figure out a/b parts! 
Then I can finish up the design accordingly and hand off to you for a final design review!

**Comment by fancyham** [comment]
Created an issue regarding adding back the footer — thanks for finding that usability problem, @fernksk :
- #1520

**Comment by staceyrebekahscott** [comment]
As per Design meeting on 01/19, waiting on updates from Content team as this will affect options for the Footer.

**Comment by staceyrebekahscott** [comment]
@fernksk A reminder to add updates from today's Design meeting here, re: what you discussed with Hannah and what is next for you to work on for Design.

**Comment by fernksk** [comment]
@staceyrebekahscott @fancyham I reached out to Hannah to review the overall design of the page and a feasibility of a "confirmation email" (the design below)

<img width="977" alt="email confirmation email draft_" src="https://user-images.githubusercontent.com/72167626/213618179-a6ce8aac-b592-470e-9378-f5d99b6e5b01.png">

- Feedback on the overall page is positive, they should be able to implement it. 
- I didn't get a clear answer on whether we're able to include image(logo) to a "confirmation email" or it'll be just a plain text email but the dev team is willing to try to make it work!! 

I guess this issue is ready for a final design review and hand-off!

The final design should be "[version B](https://www.figma.com/file/D3oq9QOXl0rFkwrEUJbABs/FOLA-Design-%231?node-id=2943%3A7818&t=WwwtKoowamnkHHG6-1)" on Figma file.

**Comment by itserindean** [comment]
@staceyrebekahscott Can you clarify what input we're waiting on from content? How have they been looped into this process? In addition to weighing in on options for the Footer, I hope they can also take a pass on the wording of the confirmation email. For example, the phrasing of "Our volunteers will review your message in 2-3 weeks" Is awkward to me.

**Comment by staceyrebekahscott** [comment]
@itserindean My comment was unclear- the content team is going to weigh in on what information is to be included in the footer, as some of the pages may be condensed down. As a result of this, where the link for this form would be in the footer depends on how many options there will be once Content shares its wireframes. 

The content team has not been looped in on the text of the email specifically, but here is a draft of a response I came up with: 

Hello (name from text field), 

Thank you for contacting Food Oasis. Our volunteers will respond to your message within (time frame- TBD). 

Below is a copy of your message: 

Date: (contacted date)
Time: (contacted time)
Subject: (from text field)
Message: (from text field) 

Note that I changed the wording from "inquiries" to "message" as I think inquiries could also be confusing. 

And we need to discuss the time frame for when responses can be expected. 2-3 weeks is a long time and not great customer service. But I am not sure what the proper time frame should be. Maybe within 7 days? Who should be responsible for responding? 

I also changed "review" to "respond", as review does not set an expectation of good customer service. But we need to think about whether that is the expectation we want to set. 

@fnkdesign Where did the decision come from to use the word "inquiries" and to use the response time frame of "2-3 weeks"? We may need to make some adjustments to that language before this is finalized, but I would like to understand the thought process.

**Comment by fernksk** [comment]
@staceyrebekahscott If I remember correctly, the content team has been looped up with the [issue 996](https://www.figma.com/file/D3oq9QOXl0rFkwrEUJbABs/FOLA-Design-%231?node-id=2943%3A7820&t=xxzPKufMPjvLVISJ-1)?  for the verification email content. So we took that as the example and discussed the usage of some portion of the automated response during the design team meeting. However, I don’t have specific reasons using “inquiries”, it just came across my mind when I thought of “questions/organization” - we can always adjust it to however the rest of the team sees fit. 

For the response time, I wasn’t sure about how much resources we have or who might be responsible for emails since we’re all volunteers here. Therefore, I though roughly 2-3 weeks would be reasonable, even though it might not be an ideal. But if we’re able to reduce the wait time, that would be great.

**Comment by staceyrebekahscott** [comment]
@itserindean As per the chat in Design team meeting, we are going to set a response time frame for messages sent to us via the contact us form to be 7 days for whatever message it is. 

I suggested we test this time frame to see how many messages we get and if the response time is reasonable. If this becomes untenable, we can change the language to something more general like "We will respond "shortly' or 'soon'." 

You and I can work out a schedule for checking the email account for these messages. 

My logic is that a time frame implies good customer service and sending a response is a personal touch that can set us apart. It is also aligned with our value proposition of having our listings validated by a person- your questions will also be answered by a person within a short time frame. 

@fernksk Let me know when you have received confirmation from the Dev team on moving forward. I want to keep track of the issue on the boards (we are still testing this workflow).

**Comment by fnkdesign** [comment]
Update/feedback from today's meeting: 

- text field has changed from "tell us about your inquiries" to "your message"
- email response time has changed to "7 days"
- additional thought/suggestion to add "create new listing" button on the FAQ page (along with "contact us" button)
- the design is final, any additional changes on "content/text" can be adjust later on
- @staceyrebekahscott the design has been brought up in the dev meeting and is ready to be implemented! 

Thank you all for support and feedback!

**Comment by fancyham** [comment]
Looks good — tagging this as ‘ready for dev lead’ to look at it and added it to their board, and to queue it up for implementing. 

@hanapotski, @entrotech  I hope I’m doing this right for your process — lemme know if not!  Please contact @fnkdesign with any questions!

**Comment by staceyrebekahscott** [comment]
On Project Management Board, moved from In Progress to In Development.

**Comment by hanapotski** [comment]
closing this in favor of https://github.com/hackforla/food-oasis/issues/1718
